### PR TITLE
Refactor presets tab to use modules as much as possible

### PR DESCRIPTION
**Pull request by chmelevskij**
This pr doesn't change any behaviour or functional parts of the code, just re-structures `presets` tab to consume modules without polluting global scope.

### CODE REVIEW

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
